### PR TITLE
Code refactoring in About and copy to clipboard share

### PR DESCRIPTION
**Pull request by Moeenahamd**

### CODE REVIEW

**Comment by vpetersson** [comment]
Thanks @Moeenahamd. We're making progress.

Here's what's missing:

- [ ] The `<title>` blocks are broken and needs to be re-instated as it was before
- [ ] The 'share` buttons for social media are broken. They need to be implemented [like this](https://codepen.io/fabienb/pen/QyBqzO) (or similar)
  - [ ] Blog posts
  - [ ] Podcast episodes 
- [ ] Code snippets on blog posts are broken (e.g. `/2021/05/07/raspberry-pi-fix.html`)
- [ ] Search is still broken
- [ ] Bullet points are not properly rendered (example at `/opnsense-and-remote-ssh.html`) - needs to support multiple nested levels
- [ ] Pagination is missing on `/blog` - you can only see the latest 15 blog posts
- [ ] The page [reading log](https://blog.viktorpetersson.com/reading-log/) hasn't been styled
- [ ] The page [remote work](https://blog.viktorpetersson.com/remote-work/) hasn't been styled
- [ ] There's a weird text snippet about video and duration here:

Screenshot 2024-03-27 at 5 25 56 PM

I think those are all issues I've found so far.

File: _layouts/podcast.html [hunk 1]
```diff
@@ -635,7 +635,7 @@
           <div class="max-sm:flex max-sm:py-3">
             <div class="share max-sm:py-0 py-4 text-white">Share:</div>
             <div class="flex max-sm:pl-3 space-x-4 max-sm:pt-2">
-              <a href="{{site.data.main.social.linkedIn}}"
+              <button onclick="share('{{site.data.main.social.linkedIn}}')"
```
[the remark this answers is not in the document]
Comment (Moeenahamd) [reply _layouts/podcast.html:638]: This does copy link to clipboard

File: _layouts/podcast.html [hunk 2]
```diff
@@ -680,7 +680,7 @@
                     d="M7.58666 0.582966L5.77778 0.580078C3.74557 0.580078 2.43227 1.92082 2.43227 3.99596V5.57092H0.61353C0.456369 5.57092 0.329102 5.6977 0.329102 5.85408V8.13601C0.329102 8.29239 0.456514 8.41903 0.61353 8.41903H2.43227V14.1771C2.43227 14.3334 2.55954 14.4601 2.7167 14.4601H5.08964C5.2468 14.4601 5.37407 14.3333 5.37407 14.1771V8.41903H7.5006C7.65776 8.41903 7.78503 8.29239 7.78503 8.13601L7.7859 5.85408C7.7859 5.779 7.75586 5.70708 7.7026 5.65395C7.64935 5.60081 7.57679 5.57092 7.50133 5.57092H5.37407V4.23581C5.37407 3.5941 5.52775 3.26834 6.36782 3.26834L7.58637 3.26791C7.74338 3.26791 7.87065 3.14113 7.87065 2.98489V0.865987C7.87065 0.709892 7.74353 0.583255 7.58666 0.582966Z"
                     fill="white"
                   /></svg></a>
-              <a href="{{site.data.main.social.twitter}}">
+              <a href="{{site.data.main.social.twitter}}"  target="_blank">
```
Comment (vpetersson) [inline _layouts/podcast.html:683]: This still just a **link** to social media. See comment in [here](https://github.com/vpetersson/blog.viktorpetersson.com/pull/27#issuecomment-2023379366).

Comment (Moeenahamd) [reply _layouts/podcast.html:683]: What shoud be there? I have seen your share reference, they redirect to social media.

Comment (Moeenahamd) [reply _layouts/podcast.html:683]: And I don't have any figma for these two pages [reading log], [remote work](https://blog.viktorpetersson.com/remote-work/)

Comment (vpetersson) [reply _layouts/podcast.html:683]: > What shoud be there? I have seen your share reference, they redirect to social media.

See my comment in the pull request. 

> And I don't have any figma for these two pages reading log, remote work

It was covered in the initial conversation. Just need a header, footer and then just regular formatting for the text on a white background.

**Pull request closed without merging**